esm-samples/jsapi-angular-cli/.browserslistrc

```diff
@@ -15,4 +15,4 @@ last 2 Safari major versions
 last 2 iOS major versions
 Firefox ESR
 not IE 9-10 # Angular support for IE 9-10 has been deprecated and will be removed as of Angular v11. To opt-in, remove the 'not' prefix on this line.
```

Comment:
Should this line be removed now that this is using Angular v11?

Reply:
Yep, I confirmed that it can be removed. https://angular.io/guide/browser-support